hcipy/util/spectral_noise.py

```diff
@@ -100,7 +100,7 @@ class SpectralNoiseFactoryFFT(SpectralNoiseFactory):
 	'''A spectral noise factory based on FFTs.
 
 	The use of FFTs means that these spectral noises will wrap at the
-	edge of the grid, ie. they are continuous from one side of the grid
+	edge of the grid, i.e. they are continuous from one side of the grid
```

Comment:
Apparently, I've been writing this wrong my whole life...

Reply:
I don't think it matters much, but who am I to contradict the Chicago manual of style  👀